src/org/exist/storage/BrokerPool.java

```diff
@@ -1004,8 +758,9 @@ protected void initialize() throws EXistException, DatabaseConfigurationExceptio
 
                             sync(broker, Sync.MAJOR);
 
+                            //TODO(AR) remove this!
                             //require to allow access by BrokerPool.getInstance();
-                            instances.put(instanceName, this);
+                            //instances.put(instanceName, this);
```

Comment:
I think how the subsystem startup in eXist done is a train-wreck, in light of what you high-lighted in `Configurator` I will give this some further thought and likely add some cleanup for that too.